l2/system-contracts/DefaultAccount.sol

```diff
@@ -37,16 +37,29 @@ contract DefaultAccount is IAccount {
 		_;
 	}
 
+	/// @notice Validates the transaction & increments nonce. 
+	/// @dev The transaction is considered accepted by the account if 
+	/// the call to this function by the bootloader does not revert 
+	/// and the nonce has been set as used.
+	/// @param _suggestedSignedHash The suggested hash of the transaction to be signed by the user.
+	/// This is the hash that is signed by the EOA by default.
+	/// @param _transaction The transaction structure itself.
+	/// @dev Besides the params above, it also accepts unused first paramter "_txHash", which
+	/// is the unique (canonical) hash of the transaction.  
 	function validateTransaction(
-		bytes32, // _txHash 
+		bytes32, // _txHash
```

Comment:
can we avoid the abbreviation and call it `transactionHash`?

Reply:
Deposit to 0x9727E868116Fdd5ba35e53ca83F07c9429Ef7F91